### PR TITLE
Fixes disk rollback

### PR DESCRIPTION
**Pull request by ashraffouda**
Signed-off-by: Ashraf Fouda <ashraf.m.fouda@gmail.com>

### CODE REVIEW

File: templates/vdisk/actions.py [hunk 1]
```diff
@@ -148,6 +138,17 @@ def save_config(job):
         if result.state != "SUCCESS":
             raise RuntimeError("Failed to save tlog conf storage: %s" % service.name)
 
+    # push nbd config to etcd
+    config = {
+        "storageClusterID": service.model.data.storageCluster,
+        "templateStorageClusterID": templateStorageclusterId,
```
Comment (khaledkbadr) [inline templates/vdisk/actions.py]: Is templateStorageClusterID default to empty string if template not found?

File: templates/vdisk/actions.py [hunk 2]
```diff
@@ -225,10 +226,14 @@ def rollback(job):
     from zeroos.orchestrator.sal.ETCD import EtcdCluster
 
     service = job.service
+    if 'vm' not in service.consumers:
+        raise j.exceptions.Input('Can not rollback a disk that is not attached to a vm')
     service.model.data.status = 'rollingback'
     ts = job.model.args['timestamp']
-
     tlogclusterconfig = get_cluster_config(job, type='tlog')
+    if not tlogclusterconfig:
```
Comment (khaledkbadr) [inline templates/vdisk/actions.py]: Can you add that check to the API as well?

Comment (ashraffouda) [inline templates/vdisk/actions.py]: this is not a bad request this should be an error 
cause tlogserver should be their anyway

Comment (khaledkbadr) [inline templates/vdisk/actions.py]: Not always, you can always create a vdisk without a tlogcluster

File: templates/vdisk/actions.py (hunk 1 above)
Comment (ashraffouda) [inline templates/vdisk/actions.py]: yeah, see the full method

File: templates/vdisk/actions.py (hunk 2 above)
Comment (ashraffouda) [inline templates/vdisk/actions.py]: you shouldn't enter if, unless there is unexpected behavior in the env

**Pull request merged**